### PR TITLE
Feat/bgd aave

### PR DESCRIPTION
**Pull request by eboadom**

### CODE REVIEW

**Review by kyzia551 — DISMISSED** [review]

**Review by kyzia551 — DISMISSED** [review]

File: src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+**The budget requested is 1’900’000 USD in stablecoins and 6’000 AAVE, 60% paid upfront and the rest over a stream of 6 months**.
+
+<br>
+
+## Specification
+
+The proposal payload does the following:
+
+1. Releases the upfront payment component to BGD Labs: 1'140'000 aDAI v2 and 3'600 AAVE.
+2. Migrates the aDAI v2 of the Aave Ethereum Collector to aDAI v3, in order to create an stream from Aave v3, following the principles of migration v2->v3 of the community.
```
Comment (sakulstra) [inline src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md:72]:
```suggestion
2. Migrates the aDAI v2 of the Aave Ethereum Collector to aDAI v3, in order to create a stream from Aave v3, following the principles of migration v2->v3 of the community.
```

File: src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+title: "Aave BGD Phase 2"
+author: "BGD Labs @bgdlabs"
+discussions: "https://governance.aave.com/t/aave-bored-ghosts-developing-phase-2/14484"
+---
+
+## Simple Summary
+
+Approval for an engagement for services between the Aave DAO and BGD Labs, focused on development related tasks and with a duration of 6 months.
+
+<br>
```
Comment (sakulstra) [inline src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md:11]: imo no so good to mix md and html

**Review by sakulstra — APPROVED** [review]

File: src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md (hunk 2 above)
Comment (eboadom) [reply src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md]: Changed

File: src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- Create aggregated workflows, by re-organizing repositories (protocol, tooling from BGD, etc). At the moment, there are a lot of developer resources scattered around, properly organizing them requires further work.
+- Development of the killswitch component approved by the community.
+- Maintain and iterate over the upcoming Aave Governance v3, a.DI and Robot systems, including:
+    - Deployment of new voting machines
+    - Support the community on assessment of new bridges, and enabling them via proposals.
+    - Fix any potential problem found.
+    - Maintain the new governance user interface.
+    - Study technical ways of increasing voting participation.
+- Batch a series of improvements for Aave v3, into an Aave v3.1 version: this will include an improvement of the liquidation mechanism, and potentially some periphery components. In addition, any bug fixing (Aave v2 and v3) would be included.
+- Support and advice on deprecation of Aave v2, involving any (reasonable) development. Items included on the Aave v3 part of the scope could potentially be applied on v2.
```
Comment (Zer0dot) [inline src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md:33]:
```suggestion
- Support and advise on deprecation of Aave v2, involving any (reasonable) development. Items included on the Aave v3 part of the scope could potentially be applied on v2.
```

File: src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+- Lead the deployment of 3 new deployment candidates (EVM), apart from the 2 pending network analyses (zkEVM, zkSync), if fully approved by the community. *Extra networks will be evaluated case-by-case*.
+
+
+**Development liaison for security, and quality assurance**
+
+- Plan and coordinate all security resources for any development project of the community., both those executed by BGD and not. This includes also the evaluation of what should be reviewed, the parties involved in it, and asking for a budget to the DAO whenever required.
```
Comment (Zer0dot) [inline src/20230828_AaveV3_Eth_AaveBGDPhase2/AaveBGDPhase2.md:39]:
```suggestion
- Plan and coordinate all security resources for any development project of the community, both those executed by BGD and not. This includes also the evaluation of what should be reviewed, the parties involved in it, and asking for a budget to the DAO whenever required.
```

**Review by Zer0dot — APPROVED** [review]

**Review by sakulstra — APPROVED** [review]

**Pull request merged**